tests/cypress/plugins/helpers.js

```diff
@@ -26,7 +26,7 @@ export const selectCourtesyCard = () => {
   cy.location("pathname").should("eq", `/eligibility/${agency.slug}`);
 
   // TODO find a more robust way to do this
```

Comment:
This TODO apt is very appropriate 😅  Ideally the code would say something like: "Look for the Courtesy Card form input text", "Select the input closest to the Courtesy Card text"

Reply:
Should create a ticket for the cooldown time to refactor this method.

Reply:
#1443